### PR TITLE
Player Display with Left and Right player navigation

### PR DESCRIPTION
**Pull request by Marthijn-B**
This screen will allow the user to navigate around the players circle and view their wonders.
And another link to bring them back to their own wonder

### CODE REVIEW

File: static/src/components/PlayerDisplay.js [hunk 1]
```diff
@@ -62,113 +57,205 @@ function mapDispatchToProps(dispatch) {
 
 export class PlayerDisplay extends Component {
 
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
+        console.log("constructor");
```
Comment (mmdonaldson) [inline static/src/components/PlayerDisplay.js]: Don't need this console.log

File: static/src/components/PlayerDisplay.js [hunk 2]
```diff
@@ -62,113 +57,205 @@ function mapDispatchToProps(dispatch) {
...
         this.state = {
-            polling: false,
+            displayID: undefined,
```
Comment (mmdonaldson) [inline static/src/components/PlayerDisplay.js]: Best practice is to initialise as null not undefined. This allows us to easily check in the render method the variable for example:
```
{ this.state.displayID && 
<h1>Hello World</h1>
}
```
Where hello world is only displayed if this.state.displayID is not null

**Review by mmdonaldson — CHANGES_REQUESTED** [review]
Hey mate can you please look at my comments then can approve

**Comment by Marthijn-B** [comment]
[UI: Show other player status](https://trello.com/c/wQ767tdM/40-ui-show-other-player-status)

File: static/src/components/PlayerDisplay.js (hunk 1 above)
Comment (Marthijn-B) [reply static/src/components/PlayerDisplay.js]: This was just me checking when the constructor was accessed. I will delete :)

**Review by mmdonaldson — APPROVED** [review]

**Pull request merged**